store/entity-store.js

```diff
@@ -161,14 +182,20 @@ window.D2L.Siren.EntityStore = {
 			if (!entity || bypassCache) {
 				const headers = new Headers();
 
-				tokenValue && headers.set('Authorization', 'Bearer ' + tokenValue);
+				const shouldAttachToken = this._shouldAttachToken(sirenLinkOrHref);
+
+				shouldAttachToken && tokenValue && headers.set('Authorization', 'Bearer ' + tokenValue);
 
 				if (bypassCache) {
 					headers.set('pragma', 'no-cache');
 					headers.set('cache-control', 'no-cache');
 				}
 
-				const request = window.d2lfetch.fetch(href, {
+				const fetch = shouldAttachToken
+					? window.d2lfetch
+					: window.d2lfetch.removeTemp('auth');
```

Comment:
BSI puts `d2l-fetch-auth` into the middleware under the name `'auth'`. What that means is even if _we_ don't attach a token, it will go ahead and do it for us, so we need to use fetch with that middleware removed.

FRAs could possibly attach said middleware under a different name, but oh well.

Reply:
All of the FRAs use the name `auth`, so I think this is good.